Exiled.Events/EventArgs/Player/TogglingNoClipEventArgs.cs

```diff
@@ -41,13 +39,13 @@ public TogglingNoClipEventArgs(Player player, bool newValue, bool isAllowed = tr
         public Player Player { get; }
 
         /// <summary>
-        ///     Gets or sets a value indicating whether or not the NoClip mode will be enabled or not.
+        ///     Gets or sets a value indicating whether or not the player can toggle the NoClip mode.
         /// </summary>
-        public bool IsEnabled { get; set; }
+        public bool IsAllowed { get; set; }
 
         /// <summary>
-        ///     Gets or sets a value indicating whether or not the player can toggle the NoClip mode.
+        ///     Gets a value indicating whether or not NoClip will be enabled after use.
         /// </summary>
-        public bool IsAllowed { get; set; }
+        public bool IsEnabled => !Player.AdminFlagsStat.HasFlag(AdminFlags.Noclip);
```

Comment:
can you remove the breaking change (it's need setter)

Reply:
OK

Reply:
are you gonne fix this ?